src/arr/trove/statistics.arr

```diff
@@ -222,9 +224,15 @@ end
 
 # please see: https://online.stat.psu.edu/stat462/
 
-fun multiple-regression(x_s_s :: List<Any>, y_s :: List<Number>) -> (Any -> Number):
+fun multiple-regression(x_s_s :: List<List<Number>>, y_s :: List<Number>) -> (Any -> Number):
   doc: "multiple-regression"
-  MR.multiple-regression(x_s_s, y_s)
+  mx_X = MX.lists-to-matrix(x_s_s.map(lam(x_s): x_s.push(1) end))
+  mx_Y = MX.lists-to-matrix(y_s.map(lam(y): [list: y] end))
+  B = MX.mtx-to-list(MX.mtx-least-squares-solve(mx_X, mx_Y))
+  fun B_pred_fn(x_s :: List<Number>) -> Number:
+    math.sum(map2(lam(b_i, x_i): b_i * x_i end, B, x_s.push(1)))
```

Comment:
Instead of making `B` be `mtx-to-list`, this feels like it ought to use `Vector`s and `dot`, or matrix-multiplication.  It's not _wrong_, it just feels lower-level than I expected.

Reply:
`vector(raw-array-from-list(the-list))` would be simpler, and cheaper, than going via an intermediate matrix

Reply:
Done. Thanks!